openfecwebapp/templates/partials/filings-filter.html

```diff
@@ -37,7 +38,33 @@
 
   <button type="button" class="js-accordion-trigger accordion__button">Filing information</button>
   <div class="accordion__content">
-    {% include 'partials/filters/amendment-indicator.html' %}
+    <div class="filter">
+      <fieldset class="js-filter js-filter-control" data-filter="checkbox" data-modifies-filter="most_recent" data-modifies-property="dataset">
```

Comment:
Do you need the `js-filter-control` class and `data-modifies-*` attributes? Those are used when a filter needs to prompt a change in another one, iirc.